Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/SamlConstants.java

```diff
@@ -135,5 +135,10 @@
     public static final String ATTRIBUTE_FRIENDLY_NAME_XUA_IACP = "Patient Privacy Policy Acknowledgement Document";
     public static final String URI_NAME_FORMAT = "urn:oasis:names:tc:SAML:2.0:attrname-format:uri";
     public static final String ADMIN_AUTH_METHOD = "urn:oasis:names:tc:SAML:2.0:ac:classes:Password";
-    
+    public static final String ADMIN_CERT_IMPORT = "importCertificate";
```

Comment:
Why these constants are part of SamlConstants. Its bette to move them to NhincConstants.